src/iosrv/tests/drpc_handler_tests.c

```diff
@@ -54,10 +54,16 @@ dummy_drpc_handler3(Drpc__Call *request, Drpc__Response **response)
 {
 }
 
+static void
+dummy_drpc_handler4(Drpc__Call *request, Drpc__Response **response)
+{
+}
+
 static drpc_handler_t handler_funcs[] = {
 		dummy_drpc_handler1,
 		dummy_drpc_handler2,
-		dummy_drpc_handler3
+		dummy_drpc_handler3,
+		dummy_drpc_handler4
```

Comment:
Useful if you add DRPC_MODULE_SRV to the multi-module tests below. Otherwise no need to add a new dummy handler.

Open question whether we want to add all the drpc modules to this test whenever we create a new one. It does serve the purpose of testing the bounds of the array inside the drpc_handler code, but I don't know if it's strictly necessary for test coverage.

Reply:
It looks every time we add a new module ID we need to add grow handler_funcs accordingly. This appears to be a "change detector" to me.

I'll add SRV to the multi-module tests.

Reply:
In theory we could just rotate the dummies to avoid adding more handlers to the list (register with handler_funcs[mod_num % 3]). Not sure how important it is that they be unique, just that there be a few different options to select.

Reply:
Just to be clear, I agree, the expansion of handler_funcs is starting to look a little pointless.